streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedStandbyTasks.java

```diff
@@ -79,7 +79,7 @@ int commit() {
             } catch (final RuntimeException e) {
                 log.error("Closing the standby task {} failed due to the following error:", task.id(), e);
             } finally {
-                removeTaskFromRunning(task);
+                removeTaskFromAllOldMaps(task, null);
```

Comment:
minor suggestion - instead of `null` what about ` Collections.emptyMap()`? This suggestion is subjective so feel free to ignore. Applies here and below.